src/scripts/records/VideoRecords.js

```diff
@@ -16,6 +16,9 @@ class Video extends ImmutableRecord({
   price: '',
   thumbnailUrl: '',
   title: '',
+  author: '',
+  free: '',
```

Comment:
I feel like we shouldn't have a `free` field. Doesn't that just mean `price === '0'`? This seems like a duplication of state and we should avoid